_people/EdwardTunstel.md

```diff
@@ -0,0 +1,12 @@
+---
+first_name: Edward
+last_name: Tunstel
+title: Associate Director and Group Leader
+school: Raytheon Technologies Research Center
+website: https://www.thehistorymakers.org/biography/edward-tunstel
+image: http://techlav.ncat.edu/images/Advisory%20Board/Ed%20Tunstel.jpg
+---
+Dr. Tunstel is an associate director of the Autonomous and Intelligent Systems Department at Raytheon, prior to which he was an Adjunct Professor for the Institute for Intelligent Systems Research and Innovation at Deakin University. His expertise is in robot navigation, fuzzy logic and autonomous behavior-based control, and applications of soft computing techniques to intelligent systems.
```

Comment:
Definitely mention JPL as his "prior to that" position

Reply:
See revisions.